extra-utils/peazip/autobuild/build

```diff
@@ -0,0 +1,43 @@
+# Set ARCH_OPTS for lazbuild's compile
+if [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    ARCH_OPTS="x86_64"
+elif [[ "${CROSS:-$ARCH}" = "arm64" ]]; then
+    ARCH_OPTS="aarch64"
+fi
+
+abinfo "Building PeaZip with Lazarus..."
+lazbuild \
+  --verbose \
+  --lazarusdir=/usr/share/lazarus \
+  --cpu=${ARCH_OPTS} \
+  --widgetset=gtk2 \
+  --max-process-count=$(nproc --all) \
+  -B "${SRCDIR}/project_pea.lpi" "${SRCDIR}/project_peach.lpi"
+
+abinfo "Installing PeaZip..."
+
+install -v -Dm755 "${SRCDIR}/peazip" "${PKGDIR}/usr/lib/peazip/peazip"
+install -v -Dm755 "${SRCDIR}/pea" "${PKGDIR}/usr/lib/peazip/pea"
+install -v -Dm644 "${SRCDIR}/../peazip_help-${PKGVER}.pdf" "${PKGDIR}/usr/share/doc/peazip/peazip_help.pdf"
+install -v -Dm644 "${SRCDIR}/FreeDesktop_integration/peazip.png" "${PKGDIR}/usr/share/pixmaps/peazip.png"
+install -v -Dm644 "${SRCDIR}/FreeDesktop_integration/peazip.desktop" "${PKGDIR}/usr/share/applications/peazip.desktop"
+
+mkdir -v "${PKGDIR}/usr/bin" -p
+pushd "${PKGDIR}/usr/bin"
+ln -sv ../lib/peazip/peazip ./peazip
```

Comment:
Save yourself some pushd... use this:

`ln -sv ../lib/peazip/peazip "$PKGDIR"/usr/bin/peazip`